third_party/rpg-maker-mv-corescript-v0/js/rpg_objects/Game_Interpreter.js

```
@@ -0,0 +1,1946 @@
+
+// Script
+Game_Interpreter.prototype.command355 = function() {
+    var startLine = this._index + 1;
+    var script = this.currentCommand().parameters[0] + '\n';
+    while (this.nextEventCode() === 655) {
+        this._index++;
+        script += this.currentCommand().parameters[0] + '\n';
+    }
+    var endLine = this._index + 1;
+    try {
+        eval(script);
```

Biome (2.5.7)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)